docs/reference/src/language/widgets/timepicker.md

```diff
@@ -0,0 +1,63 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+## struct `Time`
+
+Defines a time with hour minutes and seconds.
+
+### Fields
+
+-   **`hour`(int)**: The hour value (range from 0 to 23).
+-   **`minute`(int)**: The minute value (range from 1 to 59).
+-   **`second`(int)**: The second value (range form 1 to 59).
+
+## `TimePicker`
+
+A timer picker that is usd for selecting the time, in either 24-hour or AM/PM mode. 
+
+### Properties
+
+-   **`twenty-four-hour`**: (_in_ _bool_): Sets to true to enable 24 hour selection otherwise it is displayed in AM/PM mode.  
```

Comment:
Web browsers use the system default for [`<input type="time">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time) with regards to 12-hour format vs. 24-hour format. I think we should do the same.

If we can't determine the system default, I think we should default to 24-hours.

We could offer an optional API to override that, and then I'd call it `in property <bool> use-24-hour-format: true;` (note the default).

In reality, I'd use `in property <bool> use-24-hour-format: SlintInternals.system-time-24-hour-format;` and default to true there.

The documentation should explain what the default is here.

Reply:
It's ok for me.